IdentityServer/Infrastructure/Data/UserIdentityDbContext.cs

```diff
@@ -0,0 +1,57 @@
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Identity.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore;
+
+namespace IdentityServer.Infrastructure.Data
+{
+    public class UserIdentityDbContext : IdentityDbContext<Core.Users.UserIdentity, IdentityRole<int>, int>
+    {
+        public UserIdentityDbContext(DbContextOptions options)
+            : base(options)
+        {
+        }
+
+        protected override void OnModelCreating(ModelBuilder modelBuilder)
+        {
+            var user = modelBuilder.Entity<Core.Users.UserIdentity>()
+                .ToTable("User");
+            user.HasKey(u => u.Id);
+
+            user.HasIndex(u => u.NormalizedUserName).HasName("UserNameIndex").IsUnique();
+            user.HasIndex(u => u.NormalizedEmail).HasName("EmailIndex");
+
+            // A concurrency token for use with the optimistic concurrency checking
+            user.Property(u => u.ConcurrencyStamp).IsConcurrencyToken();
+
+            // Limit the size of columns to use efficient database types
+            user.Property(u => u.UserName).HasMaxLength(256);
```

Comment:
Really long name haha 🤣 Imagine showing 256 characters at the frontend side.

Reply:
I think that's the default AspIdentity username length, I'll change it to 25 characters.